src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java

```diff
@@ -112,14 +128,51 @@ protected void apply(final TruthVersusEval truthVersusEval, final ReadsContext r
             indelCounts.get(concordanceState).increment();
         }
 
-        if (truePositivesAndFalseNegativesVcfWriter != null && concordanceState != ConcordanceState.FALSE_POSITIVE) {
-            final VariantContext vc = annotateWithConcordanceState(truthVersusEval.getTruth(), concordanceState);
-            truePositivesAndFalseNegativesVcfWriter.add(vc);
+        if (concordanceState == ConcordanceState.TRUE_POSITIVE) {
```

Comment:
I think a switch statement may be cleaner and might communicate the fact that we're going through possible cases (i.e. enum values)